LESSON2.md

```diff
@@ -361,8 +361,3 @@ stdin.on('end', () => {
 
 *This is exactly the same as our read stream example at the top of this readme, as
 process.openStdin() is a read stream!*
-
-### Project
-
-With what we have covered in these two lessons, we now have everything we need to build some command-line tools! Move onto
```

Comment:
How about including a line like "if you're completing the course at Founders and Coders, you will cover this later"?